modular_skyrat/master_files/code/modules/client/preferences/middleware/food.dm

```diff
@@ -0,0 +1,163 @@
+// To make this system not a massive meta-pick for gamers, while still allowing plenty of room for unique combinations.
+#define MINIMUM_REQUIRED_TOXICS 2
```

Comment:
Most species only have like one toxic food, no?
This feels a little steep.

Reply:
I'll drop it down one then.